Functions/xASL_bids_CompareStructures.m

```diff
@@ -136,15 +136,17 @@
 function reportTable = xASL_bids_CompareStructures_AddEntriesToTable(reportTable,results,datasetA,datasetB)
 
     % Go through missing folders
-    if size(results.(datasetA).missingFolders,1)>0 && ~isempty(results.(datasetA).missingFolders{1})
+    if numel(results.(datasetA).missingFolders)>0 && ...
+            ~isempty(results.(datasetA).missingFolders{1})
         for iElement=1:size(results.(datasetA).missingFolders,1)
```

Comment:
Numel in the condition, but size,1 here... That might be a problem.

Reply:
Ah sure, I can change that, too. Normally the list is a column vector here, so until now it always worked, but it's easy to change that.

Reply:
Fixed here: 4d00c8f 👍 
Tested here...

```
UnitTest = xASL_ut_function_xASL_bids_CompareStructures(TestRepository);
>> UnitTest

UnitTest = 

  struct with fields:

     tests: [1×4 struct]
    passed: 1
```

... 👍